neo/Network/P2P/Capabilities/NodeCapabilities.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Neo.IO.Caching;
+
+namespace Neo.Network.P2P.Capabilities
+{
+    public enum NodeCapabilities : byte
+    {
+        [ReflectionCache(typeof(TcpServerCapability))]
+        TcpServer = 0x01,
+
+        [ReflectionCache(typeof(UdpServerCapability))]
+        UdpServer = 0x02,
+
+        [ReflectionCache(typeof(WsServerCapability))]
```

Comment:
Maybe we can use one class for `TcpServerCapability`, `UdpServerCapability`, and `WsServerCapability`?